setup.py

```diff
@@ -13,7 +13,7 @@
 
 DISTNAME = 'mne-realtime'
 DESCRIPTION = 'A module for real-time data analysis with MNE.'
-with codecs.open('README.rst', encoding='utf-8-sig') as f:
+with codecs.open('README.md', encoding='utf-8-sig') as f:
```

Comment:
You should also change the content type somewhere to say it's markdown. It might autodetect it, not sure, but better to be explicit.

Reply:
`long_description_content_type="text/markdown"` maybe?